src/js/components/Menu/__tests__/__snapshots__/Menu-test.js.snap

```diff
@@ -575,11 +575,12 @@ exports[`Menu close by clicking outside 2`] = `
 `;
 
 exports[`Menu close by clicking outside 3`] = `
-"@media only screen and (max-width:768px) {
+"@media only screen and (max-width: 768px) {
   .hYQclW {
     padding: 6px;
   }
-}"
+}
+"
```

Comment:
Can you adjust the logic on portal.js to avoid this line break on the CSS snapshot style? I don't think it is semantically accurate nor necessary and it will fix many of the snapshot changes.

Reply:
Sure. Rebased against `master` and pushed with the update.

Reply:
Perfect, thanks buddy!